src/utils/priceCalculator.js

```diff
@@ -0,0 +1,15 @@
+export const priceCalculator = (cartItem) =>
+{console.log("cartitem", cartItem);
```

Comment:
As discussed this line of code is removed in a later PR that is in the line up so it looks good to go.